flexget/manager.py

```diff
@@ -222,6 +222,15 @@ def tasks(self):
     def has_lock(self):
         return self._has_lock
 
+    @property
+    def should_reload(self):
```

Comment:
Avoid word `reload` as it is mistaken for actual file reload that is coming hopefully some day :)